google/cloud/videointelligence_v1/gapic/video_intelligence_service_client.py

```diff
@@ -213,11 +213,11 @@ def annotate_video(
             >>>
             >>> client = videointelligence_v1.VideoIntelligenceServiceClient()
             >>>
-            >>> input_uri = 'gs://cloud-samples-data/video/cat.mp4'
             >>> features_element = enums.Feature.LABEL_DETECTION
             >>> features = [features_element]
+            >>> input_uri = 'gs://cloud-samples-data/video/cat.mp4'
             >>>
-            >>> response = client.annotate_video(input_uri=input_uri, features=features)
+            >>> response = client.annotate_video(features, input_uri=input_uri)
```

Comment:
@nnegrey Are you alright with taking this change? `features` becoming required is going to break folks because it changes the parameter order. :/

Reply:
UGH, good catch. 
I think because you have to pass features every time they thought it would be a no-op, but yea that is still a bummer.

Reply:
Just sent a note to a bunch of folks to discuss this.

Reply:
Follow up: I meant to revert this manually via synthtool last year, and never got around to doing it. :) Will revert manually in this PR and follow up with a synth.py addition. Filed #7 so I don't forget about it.